lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -0,0 +1,246 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from collections import namedtuple
+from ryu.ofproto.ofproto_v1_4 import OFPP_LOCAL
+
+from .base import MagmaController
+from magma.pipelined.openflow import flows
+from magma.pipelined.openflow.magma_match import MagmaMatch
+from magma.pipelined.app.inout import INGRESS
+from ryu.lib.packet import ether_types
+from magma.pipelined.app.base import MagmaController, ControllerType
+
+GTP_PORT_MAC = "02:00:00:00:00:01"
+
+class Classifier(MagmaController):
```

Comment:
Classifier is very generic to use for a GTP specific app. Why not use GtpController as the class name and "gtp" as the APP_NAME?